docs/contributing.rst

```diff
@@ -1,4 +1,197 @@
 Contributing to AllTheBacteria
 ==============================
 
-To do
+We welcome contributions to AllTheBacteria. Please read on for instructions
+on how to contribute to the project.
+
+Background information
+----------------------
+
+Where does everything live?
+
+* Assemblies, metadata and all analysis files are hosted at the
+  AllTheBacteria project on OSF: https://osf.io/xv7q9/.
+  ie this is where we expect users of the data to go.
+* Reproducibility files (scripts, methods descriptions) are on the
```

Comment:
Reproducibility files does not seem immediately clear to me, maybe "Files to reproduce the analyses"